### DISCUSSION TITLE
Justify table column?

### DISCUSSION THREAD

**Discussion by vangberg** [marimo-team/marimo#2701] [Q&A] [answered]
Is it possible to justify the column of a `marimo.ui.table`?

**Comment by mscolnick** [comment]
Not currently - would you want this to be configurable from code as an arg to `mo.ui.table()` or from the UI or both?

**Reply by vangberg** [reply]
Mostly from the code, I think.

**Reply by mscolnick** [reply]
just created a ticket https://github.com/marimo-team/marimo/issues/2718

**Comment by akshayka** [comment, marked as answer]
@vangberg, this is now possible using `text_justify_columns`, a dictionary from column names to one of `"left"`, "center", or "right": https://docs.marimo.io/api/inputs/table.html.

**Reply by vangberg** [reply]
Perfect, thank you!
